functions.lua

```diff
@@ -66,6 +66,50 @@ postool.hasMeseconsDebug = function()
 end -- hasMeseconsDebug
 
 
+-- return a string to show on HUD containing info about biome that player is standing on
+-- thanks to Montandalar for initial method
+if postool.hudBiomeVerbose then
```

Comment:
this check should be done inside the function then wouldn't need to have two functions to maintain.